app/routers/v2.py

```diff
@@ -14,14 +14,10 @@ class Sources(str, enum.Enum):
     A source available for retrieving data.
     """
 
-    JHU = "jhu"
-    CSBS = "csbs"
-    NYT = "nyt"
-
```

Comment:
We already have an enum defined.

I agree with the essential direction of the PR though. To actually use the enum.

Maybe we ought to define it in `app/data`.